File: Selenium/Activities/SeleniumTest.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.firefox.service import Service

# Start the Driver
with webdriver.Firefox() as driver:
  
  # Open the browser to the URL
  driver.get("https://training-support.net")
  print("page title is " + driver.title)

  driver.get()
  driver.back()
  driver.forward()
  driver.refresh()
  
  # Perform testing and assertions
  ...
  
  # Close the browser
  # Feel free to comment out the line below
  # so it doesn't close too quickly
  driver.quit()

File: Selenium/Activities/SeleniumTest.java
package activities;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class SeleniumTest {

	public static void main(String[] args) {
		
		//initializing a driver
		WebDriver driver = new FirefoxDriver();
		
		//open a browser page
		driver.get("https://training-support.net");
		
		//interactions
		
		//print page title
		System.out.println(driver.getTitle());
		
		//close
		//closes entire browser
		//driver.quit(); 
		
		//closes currently active tab
		driver.close(); 
		
	}

}
